cvmfs/mountpoint.cc

```diff
@@ -1783,6 +1792,19 @@ bool MountPoint::SetupBehavior() {
   LogCvmfs(kLogCvmfs, kLogDebug, "kernel caches expire after %d seconds",
            static_cast<int>(kcache_timeout_sec_));
 
+  uint64_t statfs_time_cache_valid = 0;
+  if (options_mgr_->GetValue("CVMFS_STATFS_CACHE_TIMEOUT", &optarg)) {
+    // 0ul does not work because of ubuntu 18
+    if ( optarg.size() > 0 ) {
+    statfs_time_cache_valid =
+      std::max(static_cast<uint64_t>(0),
```

Comment:
I think we don't need it: the second argument is already unsigned (`String2Uint64`), so it must be >=0.